### PR TITLE
bump dependencies

### PR DESCRIPTION
**Pull request by jasisk**
Leaving express as-is because of the peerDep.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,40 +55,40 @@
...
+    "lusca": "^1.0.3",
+    "meddleware": "^3.0.1",
+    "morgan": "^1.5.2",
+    "serve-favicon": "^2.2.0",
+    "serve-static": "^1.9.2",
+    "shortstop": "^1.0.1",
     "shortstop-handlers": "^1.0.0",
     "shortstop-resolve": "^1.0.1",
-    "shush": "~0.0.1"
+    "shush": "0.0.1"
```
Comment (thefourtheye) [inline package.json]: Didn't you mean `^0.0.1`?

**Comment by aredridel** [comment]
LGTM, needs rebase?

**Comment by jasisk** [comment]
re:baseball:d

**Comment by aredridel** [comment]
Yup, still LGTM

**Comment by pvenkatakrishnan** [comment]
:+1:

**Comment by jasisk** [comment]
I'm trying to make another owner appear ...
abracatabra

**Comment by pvenkatakrishnan** [comment]
waiting for  travis.. seems to be on vacation today

**Pull request merged**